Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG TARGETARCH
 WORKDIR /namespace-lister
 
 # Copy the Go Modules manifests
-COPY go.mod go.sum .
+COPY go.mod go.sum ./
```

Comment:
I don't see a difference in behavior - why is this needed?

Reply:
I applied a fix for a complaint from my linter. When multiple files are copied, it's best practice to explicitly declare a folder.

Reply:
Then maybe we should have something like this?
```suggestion
COPY go.mod go.sum /namespace-lister/
```

Reply:
we are already setting the WORKDIR to `/namespace-lister`. I'd prefer this to remain relative to the WORKDIR